src/index.ts

```diff
@@ -643,18 +643,15 @@ export const createInstance = (config: Config): Instance => {
           tokenAddress
         );
 
-        let approvalAmount = ethers.constants.MaxUint256;
-        if (amount) {
-          approvalAmount = ethers.utils.parseEther(amount);
```

Comment:
`parseEther` is converting into `wei` unit of 18 decimals by default.
Can I confirm that amount in the paramter is in `wei` unit?